docs/getting-started.md

```diff
@@ -1,21 +1,168 @@
+
+![Screenshot Showing Reply](getting-started-imgs/reply.png)
+
+### Thumbs-up Reaction
+
+If you want to show you agree with the current message, type <kbd>+</kbd> to add a thumbs-up reaction. Type <kbd>+</kbd> again to remove the thumbs-up.
+
+![Screenshot Showing Thumbs-up](getting-started-imgs/thumbs-up.png)
+
+## Send a PM
+
+Let's try sending a PM to the author of a message. Select the message you sent to the [# test here](https://chat.zulip.org/#narrow/stream/7-test-here) stream earlier and press <kbd>R</kbd> to send a PM to yourself. Type your message in the message editor that appears at the bottom of the middle column and then type <kbd>ctrl</kbd><kbd>d</kbd> to send. Press <kbd>P</kbd> to go to your PM's and make sure everything worked properly.
+
+You can also send a PM by moving your cursor to the list of "Users" in the left column and selecting the name of the person you'd like to send a message to.
+
+### Send a PM with the New Autocomplete Feature
+
+If you grabbed the latest unreleased version from [Github](https://github.com/zulip/zulip-terminal) or are running a version greater than 0.6.0 (use `zulip-term --version` to check), you can try out our new autocomplete feature! 
+
+To send a PM using the autocomplete feature:
+1. Type <kbd>x</kbd> from anywhere in Zulip Terminal. A message editor will pop open at the bottom of the middle column. 
+2. Type in part of the name of the person you'd like to send a PM to (IMG 1)
+3. Press <kbd>ctrl</kbd><kbd>f</kbd> and a list of potential recipients will appear in the footer (highlighted in red). 
+4. Press <kbd>ctrl</kbd><kbd>f</kbd> until the name of the person you want to send a message to is highlighted (IMG 2). 
+5. Press <kbd>tab</kbd> to jump to the message section and type in your message. 
+6. Press <kbd>ctrl</kbd><kbd>d</kbd> to send.
+7. Press <kbd>esc</kbd> to get out of the message editor.
+
+Try following the steps above to send a message to the Welcome Bot!
+
+*You must use <kbd>ctrl</kbd><kbd>f</kbd> to autocomplete the name and zulip address of the person you want to PM. If you just type in their full name Zulip Terminal will throw an error.*
+
+![Screenshot Showing First Step to Send PM](getting-started-imgs/send-PM1.png)
+![Screenshot Showing Second Step to Send PM](getting-started-imgs/send-PM2.png)
+
+## Create a New Topic
+
+To create a new topic in a stream, type <kbd>c</kbd> and a message editor will pop open at the bottom of the middle column. Make sure your cursor is in the middle column, otherwise nothing will happen when you type <kbd>c</kbd>.
```

Comment:
Not pertinent to the tutorial right now, but do you think we should support this working in the stream or topic list?

Reply:
I do feel like it's a bit confusing how it works in the middle column but nowhere else. Considering how the c key isn't repurposed for something else in other sections, I think it would be nice if the c key was consistent across ZT